msal4j-brokers/pom.xml

```diff
@@ -30,7 +30,12 @@
         <dependency>
             <groupId>com.microsoft.azure</groupId>
             <artifactId>msal4j</artifactId>
-            <version>1.13.2</version>
+            <version>1.13.3</version>
+        </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>javamsalruntime</artifactId>
+            <version>1.0.0</version>
```

Comment:
Commenting to ensure we correct the version number here once interop's beta version is published to maven.

Reply:
In the latest commit I changed this to [0.13.3,) . This will start with version 0.13.3 of the interop (the current MSALRuntime version), and will get the latest version from then on (so that we don't need to update the broker package when we update the interop)